src/Collisions/babylon.collisionCoordinator.ts

```diff
@@ -354,6 +354,8 @@ module BABYLON {
             collider.retry = 0;
             collider.initialVelocity = this._scaledVelocity;
             collider.initialPosition = this._scaledPosition;
+            collider._collisionMask = excludedMesh && excludedMesh._collisionMask ? excludedMesh._collisionMask : collider._collisionMask;

```

Comment:
no need to add this on the collider. The excluded mesh is sent to the _collideWithWorld